CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+## [2.1.0] - 2022-04-29
+
+### Added
+- Added the workloadapi.WatchX509Bundles method which watches X.509 bundles from the Workload API (#192)
```

Comment:
We usually don't include the "Added", "Changed", etc. word to the start of the phrase, as it's already included in the title of the section.